### DISCUSSION TITLE
How to use the `GoDecl` in  the `vim-go` plugin

### DISCUSSION THREAD

**Discussion by SimFG** [ayamir/nvimdots#554] [General]
![image](https://user-images.githubusercontent.com/21985684/223301515-f568b2b3-7ebf-4704-b86d-858b48b1d8bf.png)
I found there is the `telescope-fzf-native.nvim` plugin in the plugin list. What do I need to do? Thanks

**Comment by aarnphm** [comment]
Doesn't seem like this is a telescope-fzf-native.nvim deps right?

**Reply by SimFG** [reply]
yes, it need the fzf.vim

**Reply by charliie-dev** [reply]
so install one of those.

**Reply by SimFG** [reply]
the `fzf.vim` can work in the nvim? I don't know how to install it by the `lazy.nvim` plugin manager, because i don't find the document in the `fzf.vim` repo.

**Reply by aarnphm** [reply]
you can install any arbitrary vim/lua plugins

**Reply by SimFG** [reply]
thanks, i will be a try!!!

**Comment by SimFG** [comment]
@aarnphm how can i success to install it
![image](https://user-images.githubusercontent.com/21985684/224481375-8b22888a-88c3-4f7e-a656-6074469c2b96.png)

**Comment by ayamir** [comment]
Solved at https://github.com/ayamir/nvimdots/issues/563.
